campaignion_expiry/src/ContactCron.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Drupal\campaignion_expiry;
+
+use Drupal\campaignion\Contact;
+
+/**
+ * Cron-job for expiring old contact records.
+ */
+class ContactCron {
+
+  /**
+   * Create a new cron-job instance.
+   */
+  public function __construct(int $time_limit, string $inactive_since_str, bool $keep_mp_fields = FALSE) {
```

Comment:
It would be nice to document the construction arguments (or use class variables, like SubmissionCron).